redaxo/src/core/lib/console/command_disable_packages.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/**
+ * @package redaxo\core
```

Comment:
```suggestion
 * Disable loading(bootstrapping) of packages when the command is executed. This might be useful e.g. when loading of a package requires a db-connection, but connection credentials have not been defined yet.
 *
 * @package redaxo\core
```